certify-service/src/main/resources/application-local.properties

```diff
@@ -94,12 +94,12 @@ mosip.certify.key-values={\
               'credential_issuer': '${mosipbox.public.url}',   \
               'credential_endpoint': '${mosipbox.public.url}${server.servlet.path}/issuance/credential', \
               'display': {{'name': 'Insurance', 'locale': 'en'}},\
-              'credentials_supported' : { \
+              'credential_configurations_supported' : { \
                  "InsuranceCredential" : {\
                     'format': 'ldp_vc',\
                     'scope' : 'sunbird_rc_insurance_vc_ldp',\
                     'cryptographic_binding_methods_supported': {'did:jwk'},\
-                    'cryptographic_suites_supported': {'Ed25519Signature2020'},\
+                    'credential_signing_alg_values_supported': {'Ed25519Signature2020'},\
                     'proof_types_supported': {'jwt'},\
```

Comment:
As per draft 13, this is changed to below format and we have include all the algorithms that we support as per https://github.com/mosip/mosip-config/blob/develop/esignet-default.properties#L455

            "proof_types_supported": {
                "jwt": {
                    "proof_signing_alg_values_supported": [
                        "ES256"
                    ]
                }
            },

Reply:
I believe this also requires some code changes, since I was able to find reference of proof_types_supported used within the code